packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -337,6 +343,7 @@ abstract class BaseFlutterTask extends DefaultTask {
     String targetPath
     @Optional @Input
     Boolean previewDart2
+    Boolean trackWidgetCreationLocations
```

Comment:
Add `@Input`. Otherwise changing this value will not cause Gradle to rebuild.
It probably needs to be `@Optional` too.

Reply:
Thanks! That is what I was missing. rebuilds now work as expected.